system-int-test/test/cmr/system_int_test/search/umm_json_search_test.clj

```diff
@@ -1,16 +1,17 @@
 (ns cmr.system-int-test.search.umm-json-search-test
   "Integration test for UMMJSON format search"
```

Comment:
I know this docstring was there from before ... don't we hyphenate this? UMM-JSON?